sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -128,6 +129,37 @@ export class InstitutionDto extends PartialType(CreateInstitutionDto) {
   }
 }
 
+export class AESTInstitutionDetailDto {
+  legalOperatingName: string;
+  operatingName: string;
+  primaryPhone: string;
+  primaryEmail: string;
+  website: string;
+  regulatingBody: string;
+  institutionTypeName: string;
+  formattedEstablishedDate: string;
+  primaryContactEmail: string;
+  primaryContactFirstName: string;
+  primaryContactLastName: string;
+  primaryContactPhone: string;
+  legalAuthorityEmail: string;
+  legalAuthorityFirstName: string;
+  legalAuthorityLastName: string;
+  legalAuthorityPhone: string;
+  address: {
+    addressLine1: string;
+    addressLine2: string;
+    city: string;
+    country: string;
+    provinceState: string;
+    postalCode: string;
+  };
+}
+
+export interface BasicInstitutionInfo {
```

Comment:
In BaseDTO we can have more properties which are common across APIs so that it can be effectively used.

Reply:
Yeah that was the idea, for now keeping only the required field